composer.json

```diff
@@ -24,7 +24,7 @@
     ],
     "homepage": "http://moneyphp.org",
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "^8.1",
```

Comment:
Please keep the strategy we used before, explicit support of versions. Just add `|| ~8.4.0`

Reply:
But then I have to do this every year. It's so annoying, like I stated in the OP. This is one of the few packages that does this. Why.

Reply:
Even the creator, Composer, doesn't do this: https://github.com/composer/composer/blob/e4038e5e3bffe22cb815dc1663ea76be96d4222e/composer.json#L25
Or PHPStan https://github.com/phpstan/phpstan-src/blob/2251258cdac7ac0aa36cdf88e5924780bc230f70/composer.json#L8

Reply:
I don't think it's annoying. Especially with people looking for early support of newer PHP versions, like yourself, I find it useful that I know for sure that this package is delivering full-support for it. Moreover, it creates a moment where we can reconsider previous decisions. Maybe it's overly conservative, I am fine with that.